src/boot/efi/bcd.c

```diff
@@ -1,8 +1,28 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
-#include <efi.h>
-#include "macro-fundamental.h"
-#include "util.h"
+#ifdef SD_BOOT
+#        include <efi.h>
```

Comment:
We use two spaces for indentation of `#  include` and `#  define`.

Reply:
Does clang-format really not have an option for this? *sigh*